hvpy/utils.py

```diff
@@ -75,3 +77,53 @@ def create_layers(layer: list) -> str:
     '[3,1,50],[10,1,50]'
     """
     return ",".join([_create_layer_string(_to_datasource(s), o) for s, o in layer])
+
+
+def _to_event_type(val: Union[str, EventType]) -> EventType:
+    """
+    Validates the input and converts it to a EventType enum.
+    """
+    # This is an undocumented attribute of Enums
+    if isinstance(val, str) and val in EventType._value2member_map_:
```

Comment:
We might not want to do this but its really handy.

Reply:
Interesting find, I'll allow it.

Reply:
What does `_value2member_map_` do?